### ISSUE TITLE
Relai is closed source

### ISSUE THREAD

**Issue by danielabrozzoni** [RiccardoMasutti/DigitalSP#1]
I think you need to mention that Relai doesn't publish the code source, hence is not advised to keep money there (I think you should buy and move immediately, or at least move occasionally, even if it's a bit expensive fee-wise).
I wouldn't bother normally, but I understand that this thread has in mind privacy and security, and putting lots of money on a god_knows_what_it_does app is not really safe :)

**Comment by RiccardoMasutti** [comment]
I agree with you, unfortunately Relai it's closed source but I still think that it's a great company/app for new european bitcoiners because it allows them to buy KYC-free bitcoins.

Also, Relai allows you to buy to external wallet: https://support.relai.ch/buy-and-sell-bitcoin/buy-to-external-hardware-wallet-electrum/

**Comment by danielabrozzoni** [comment]
I'm not saying you shouldn't put it in the list. I'm saying that since it's a list that has security in mind as well, you should put it but suggesting to buy to external wallet, or at least not to keep to many sats on it.
